INFERENCE.md

```diff
@@ -0,0 +1,202 @@
+# Inference tips
+
+Parler-TTS benefits from a number of optimizations that can make the model up to 4 times faster. Add to this the ability to stream audio as it's being generated, and you can achieve time-to-first-audio in under 500ms on a modern GPU.
+
+## 📖 Quick Index
+* [Efficient Attention Implementation](#efficient-attention-implementations)
+* [Compilation](#compilation)
+* [Streaming](#streaming)
+* [Batch generation](#batch-generation)
+
+## Efficient Attention implementations
+
+Parler-TTS supports [SDPA](https://pytorch.org/docs/master/generated/torch.nn.functional.scaled_dot_product_attention.html) and [Flash Attention 2](https://github.com/Dao-AILab/flash-attention).  
+
+SDPA is used by default and speeds up generation time by up to 1.4 times compared with eager attention.
+
+To switch between attention implementations, simply specify `attn_implementation=attn_implementation` when loading the checkpoints:
+
+```py
+from parler_tts import ParlerTTSForConditionalGeneration
+
+torch_device = "cuda:0"
+torch_dtype = torch.bfloat16
+model_name = "parler-tts/parler-tts-mini-v1"
+
+attn_implementation = "eager" # "sdpa" or "flash_attention_2"
```

Comment:
let's make `sdpa` the default since it is faster?

Reply:
`sdpa` is used by default, we're showing here how to use other implementations!